common/chain/chaincfg.go

```diff
@@ -129,10 +129,10 @@ type VersionParams struct {
 
 type MigrationParams struct {
 	// StartHeight is the height from which the state from the old chain is to be migrated.
-	StartHeight int64 `json:"start_height"`
+	StartHeight int64 `json:"start_height,omitempty"`
```

Comment:
Oh yeah this won't mess anything up like a nil struct ptr but is less confusing :+1:

Reply:
Yeah I went to go make a nil struct pointer but I am finding a _ton_ of places where we might have dereference errors